oprf/oprf.go

```diff
@@ -243,7 +228,7 @@ func mustWrite(h io.Writer, data []byte) {
 }
 
 // FinalizeHash computes the final hash for the suite.
-func finalizeHash(s *suite, data, iToken, info, context []byte) []byte {
+func (s *suite) finalizeHash(data, iToken, info []byte) []byte {
```

Comment:
Mmm.. why do we make this a method of the suite if it is more an auxiliary func?

Reply:
either way can work, having it as a method reduces the number of parameters passed to it